packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-mixed.twig

```diff
@@ -78,7 +79,9 @@
     {% endif %}
   </div>
 
-  <nav class="ma__header__nav" aria-label="main navigation" id="main-navigation" role="navigation">
+  {# Unable to use aria for labeling since inconsistent support for the tags by screen readers as of Feb. 22, 2023. #}

```

Comment:
I think aria-label is fine to use on navigation, see https://www.davidmacd.com/blog/does-aria-label-override-static-text.html